tests/cross_functional/system_test/test_object_expiration_system.py

```
@@ -383,6 +385,21 @@ def check_if_objects_expired(mcg_obj, bucket_name, prefix=""):
+        logger.info("Waiting for NooBaa to stabilize after disruptions...")
+        ceph_cluster = CephCluster()
+        ceph_cluster.wait_for_noobaa_health_ok()
+        logger.info("NooBaa health check passed after disruptions")
+
+        # Perform ceph health check after disruptions
+        try:
+            ceph_health_check(
+                namespace=config.ENV_DATA["cluster_namespace"], tries=45, delay=60
+            )
+            logger.info("Ceph health check passed after disruptions")
+        except Exception as ex:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)